### ISSUE TITLE
[Tutorial] How to use this dependency on React Native 0.60

### ISSUE THREAD

**Issue by ebellumat** [tinycreative/react-native-intercom#311]
I finded a way to install this depedency on React Native 0.60.4 by myself.

Don't run react-native link for this dependency.
Don't try to install manually.

Follow these steps and only configure the AppDelegate.m

Put this line on your pod file and run pod install

```pod 'react-native-intercom', :path => '../node_modules/react-native-intercom'```

After that, instead put this import ```#import "Intercom/intercom.h"``` in AppDelegate.m, put this import ```#import <Intercom/intercom.h>``` to use this dependency from Pod

I Hope this helps, folks.

**Comment by MuhammadNayabButt** [comment]
@ebellumat can you show your pod file. I am new to react so my question is
do we need to add `pod 'Intercom'` in to our project to install native library in iOS and then we can use `pod 'react-native-intercom', :path => '../node_modules/react-native-intercom'`

**Comment by Subway19** [comment]
https://github.com/tinycreative/react-native-intercom/issues/317

**Comment by waclock** [comment]
I'm also facing issues installing this using react-native `0.61.2`.

Manually linking/installing the library ends up in 
` fatal error: 'React/RCTBridge.h' file not found`

Also tried this approach, and the app compiles, however I get this 
![Screenshot 2019-11-06 13 24 13](https://user-images.githubusercontent.com/2390687/68316750-bb8f9680-0098-11ea-93ad-369c84b9c661.png)
and intercom's docs still say 'Intercom isn’t installed for your iOS app yet'

**Comment by ljcp** [comment]
@ebellumat thank you , this should be the correct way to install it , manual install throws recursive header expansion https://github.com/tinycreative/react-native-intercom/issues/320

**Comment by MayoudP** [comment]
When adding `pod 'react-native-intercom', :path => '../node_modules/react-native-intercom'` to my podfile, I'm getting this error : 

CocoaPods could not find compatible versions for pod "React/Core":
  ```
In Podfile:
    react-native-intercom (from `../node_modules/react-native-intercom`) was resolved to 13.0.0, which depends on React/Core
```

If I'm looking into my `node-modules/react-native-intercom`, the file `react-native-intercom.podspec` is right here...

Any ideas how to fixed it ?

**Comment by darajava** [comment]
Thank you!! This completely solved my issue and I wouldn't have wasted 6 hours if this was in the documentation!!!
